.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+align = true
+assumeStandardLibraryStripMargin = true
+maxColumn = 100
```

Comment:
can we have it as 120?

Reply:
I also find this one useful:
https://scalameta.org/scalafmt/docs/configuration.html#includecurlybraceinselectchains

```
List(1)
  .map { x =>
    x + 2
  }
  .filter(_ > 2)
```

All operations get aligned by the ., so we can clearly see all the transformations being applied.

Reply:
> can we have it as 120?

Sure.

> I also find this one useful:
> [scalameta.org/scalafmt/docs/configuration.html#includecurlybraceinselectchains](https://scalameta.org/scalafmt/docs/configuration.html#includecurlybraceinselectchains)

It's true by default.